woocommerce/woocommerce
# Use latest PHP version to run Travis build jobs

### Changes proposed in this Pull Request:

This PR updates Travis configuration to use the latest PHP version to run build jobs for e2e tests, coding standard checks, unit test code coverage and for running unit tests using WP nightly. For those jobs we
want to always use the latest PHP version.

### How to test the changes in this Pull Request:

1. Check the Travis configuration changes.
2. Make sure that the Travis build is still passing.
